### DISCUSSION TITLE
Allow smtp auth only from one IP for a particular user/email

### DISCUSSION THREAD

**Discussion by ck-78** [modoboa/modoboa#2269] [Help]
Hello,

I successfully setup a mail server using the modoboa installer (1.17.0). I added a domain and several mailboxes. No I just want to add a special ("technical") account for a webserver. The webserver should be able to send emails through this account. The goal is to allow the smtp authentication for this account only from the static webserver IP. All other accounts should be able to login from everywhere.
I found the allow_nets extra field possibility in the dovecot documentation. But I can´t find a configuration example for SQL authentication of modoboa. There is no such field in the user table of modoboa.
Any suggestions?
Thanks a lot for your help.
Christian

**Comment by H-A-L-I-S** [comment]
SMTP is handled by postfix. You can check, if there is a option for that. http://www.postfix.org/postconf.5.html

IMAP is handled by dovecot. You can check their documentation, if there is also a option for that: https://www.dovecot.org/documentation

If you mean the modoboa-webmail, no clue. Why would you anyways do all that effort, if you can just use a secure password to protect the admin account instead?
